assembl/views/api/post.py

```diff
@@ -669,7 +669,7 @@ def create_post(request):
             locale = discussion.main_locale
         # print subject
         if subject is not None and len(subject):
-            new_subject = "Re: " + restrip_pat.sub('', subject).strip()
```

Comment:
@n0izn0iz I disagree. The old interface should stay unchanged. I think you should revert this commit and remove the 'Re: ' in javascript.

Reply:
Ok, I reverted in https://github.com/assembl/assembl/commit/f43d413412f2e9ad27c4242ced6e6e1d213d82e8 and did it in js https://github.com/assembl/assembl/commit/6e9a20adfc30545605d149205ba2264aa0f51c6d